tests/ga/test_remoteaccess_handler.py

```diff
@@ -98,14 +121,21 @@ def test_add_user_bad_password_data(self, _1, _2, _3):
         tstuser = "foobar"
         expiration_date = datetime.utcnow() + timedelta(days=1)
         pwd = tstpassword
-        rah.add_user(tstuser, pwd, expiration_date)
+        error = "Error adding user {0} cleanup successful\nInner error: test exception for bad password".format(tstuser)
+        if is_python_version_36():
```

Comment:
Not a huge fan of this type of check. We have started adding version agnostic helpers to `tests/tools.py`, for example see #1266 which adds `assertListEqual` if missing. Would you mind doing the same for `assertRaisesRegex`?

Reply:
I actually already did this, but being that with Python 2.6 doesn't support the assert regex so that part is skipped, I thought this was better since it clearly indicates to anyone lookin in the future what's going on.

---
In reply to: [207063999](https://github.com/Azure/WALinuxAgent/pull/1281#discussion_r207063999) [](ancestors = 207063999)

Reply:
... never mind, I can just do the regex myself.  Done.

---
In reply to: [207063999](https://github.com/Azure/WALinuxAgent/pull/1281#discussion_r207063999) [](ancestors = 207063999)